sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero.hpp

```diff
@@ -270,6 +285,7 @@ struct _ur_device_handle_t : _pi_object {
   // Whether to use immediate commandlists for queues on this device.
   // For some devices (e.g. PVC) immediate commandlists are preferred.
   bool ImmCommandListsPreferred;
+  bool isImmedidateCommandListUsed = false;

```

Comment:
I suggest that we fuse these 2 booleans into just one.

Reply:
thanks @smaslov-intel . good idea. changing it.